<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shortcut</class>
 <widget class="QDialog" name="shortcut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcut</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_key">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxVisibleItems">
      <number>64</number>
     </property>
     <item>
      <property name="text">
       <string>F1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Home</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PgUp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PgDown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Up</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Down</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tab</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Backspace</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_action">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxVisibleItems">
      <number>15</number>
     </property>
     <item>
      <property name="text">
       <string>Back</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BreakTimer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Forward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Loop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NextTrack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PlayPause</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PrevTrack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Redo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rewind</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Timer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Undo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VolumeDown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VolumeUp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zero</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_ctrl">
     <property name="text">
      <string>Ctrl</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_shift">
     <property name="text">
      <string>Shift</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_alt">
     <property name="text">
      <string>Alt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>shortcut</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>shortcut</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
